src/app/index.config.coffee

```diff
@@ -52,6 +52,19 @@
     )
   )
 
+  .config ($httpProvider) ->
+    $httpProvider.interceptors.push ($q, $injector, DEVISE_CONFIG, $rootScope) ->
+      {
+        request: (config) ->
+          # Intercept requests made to the API and reset the sessions timeout
+          if DEVISE_CONFIG.timeout_in > 0
+            if config.url.includes('mnoe') && !config.url.match(/(sign_out|sign_in)/) && !$rootScope.timeoutSet
+              $rootScope.timeoutSet = true
```

Comment:
I don't understand what you're doing here?
We wan't to reset the timer everytime there's a request to the API?